ballerina/tests/test.bal

```diff
@@ -36,7 +38,30 @@ ConnectionConfig sfConfig = {
     }
 };
 
-Client baseClient = check new (sfConfig);
+ConnectionConfig sfConfigPasswordFlow = {
+    baseUrl: baseUrl,
+    auth: {
+        password,
+        username,
+        tokenUrl: refreshUrl,
+        clientId: clientId,
+        clientSecret: clientSecret,
+        credentialBearer: "POST_BODY_BEARER"
+    }
+};
+
+ConnectionConfig sfConfigCredentialsFlow = {
+    baseUrl: baseUrl,
+    auth: {
+        clientId: clientId,
+        clientSecret: clientSecret,
+        tokenUrl: refreshUrl
+    }
+};
+
+Client baseClient = check new (sfConfigRefreshFlow);
+Client baseClientPasswordFlow = check new (sfConfigPasswordFlow);
+Client baseClientCredentialsFlow = check new (sfConfigCredentialsFlow);
```

Comment:
same as above. Also we don't need to use the `base` prefix in here IMO.

Reply:
https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/308